exotica_core/include/exotica_core/kinematic_element.h

```diff
@@ -121,6 +121,8 @@ class KinematicElement
     KDL::Frame generated_offset = KDL::Frame::Identity();
     bool is_trajectory_generated = false;
     std::vector<double> joint_limits;
+    std::vector<double> velocity_limit;
```

Comment:
We don't need a vector here - it's a single quantity. The upper and lower limits are the same.

Reply:
If this should be a double, then why did you want to move it out of the joint_limits vector? I thought you wanted to make it compatible with quaternions in the future.